common/src/main/java/com/redhat/cloud/notifications/XRhIdentityUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.redhat.cloud.notifications;
+
+import io.vertx.core.json.JsonObject;
+
+public class XRhIdentityUtils {
+
+    /**
+     * Generates a base64 encoded identity header contents, which just contains
+     * the specified account id and organization id.
+     * @param accountId the EBS account number to place in the contents.
+     * @param orgId the organization id to place in the contents.
+     * @return a base64 encoded string.
+     */
+    public static String generateEncodedXRhIdentity(final String accountId, final String orgId) {
+        final JsonObject identity = new JsonObject();
+
+        if (accountId != null && !accountId.isBlank()) {
+            identity.put("account_number", accountId);
+        }
```

Comment:
Is the `account_number` field needed in Sources? Otherwise, it'd be great if we could remove it here because that data may be removed from the `endpoints` table in the DB soon.

Reply:
Sure, let's do that! Sources also prioritizes the organization ID, so I?m all for removing the account number.